src/libs/EmojiUtils.tsx

```diff
@@ -662,6 +662,11 @@ function containsOnlyCustomEmoji(text?: string): boolean {
     return privateUseAreaRegex.test(text);
 }
 
+function isSingleEmojiLine(line: string) {
```

Comment:
```suggestion
function isEmojiOnSeparateLine(line: string) {
```

Here as well - I think this name would be a bit more clearer